src/ansys/units/quantity.py

```diff
@@ -71,18 +71,20 @@ def __init__(self):
 
 class Quantity:
     """
-    A class containing a physical quantity's value and associated units.
+    A class representing a physical quantity's value and associated units.
```

Comment:
I think we want to make this change on all the other class descriptions that use the word "containing" as well as reword it on anything that does not.  i.e. `UnitSystem`, `UnitRegistry`, `Unit`, `QuanitityMap`, `Dimensions` all come to mind here.

Reply:
Updated on the other classes.